src/@lekoarts/gatsby-theme-minimal-blog/components/navigation.tsx

```diff
@@ -1,38 +1,56 @@
 /** @jsx jsx */
-import * as React from "react"
-import { jsx } from "theme-ui"
-import { Link } from "gatsby"
-import useMinimalBlogConfig from "../hooks/use-minimal-blog-config"
-import replaceSlashes from "../utils/replaceSlashes"
+import * as React from "react";
+import { jsx } from "theme-ui";
+import { Link } from "gatsby";
+import useMinimalBlogConfig from "../hooks/use-minimal-blog-config";
+import replaceSlashes from "../utils/replaceSlashes";
+import { useI18next } from "gatsby-plugin-react-i18next";
 
 type NavigationProps = {
   nav: {
-    title: string
-    slug: string
-  }[]
-}
+    title: string;
+    slug: string;
+  }[];
+};
 
 const Navigation = ({ nav }: NavigationProps) => {
-  const { basePath } = useMinimalBlogConfig()
+  const { basePath: basePath_ } = useMinimalBlogConfig();
+  const { language, i18n } = useI18next();
+  const isDefaultLanguage =
+    Array.isArray(i18n.options.fallbackLng) &&
+    i18n.options.fallbackLng?.includes(language);
+  const basePath = isDefaultLanguage
+    ? `/${basePath_}`
+    : `/${basePath_}/${language}`;
 
   return (
     <React.Fragment>
       {nav && nav.length > 0 && (
-        <nav sx={{ "a:not(:last-of-type)": { mr: 3 }, fontSize: [1, `18px`], ".active": { color: `heading` } }}>
-          {nav.map((item) => (
-            <Link
-              activeClassName="active"
-              key={item.slug}
-              sx={(t) => ({ ...t.styles?.a })}
-              to={replaceSlashes(`/${basePath}/${item.slug}`)}
-            >
-              {item.title}
-            </Link>
-          ))}
+        <nav
+          sx={{
+            "a:not(:last-of-type)": { mr: 3 },
+            fontSize: [1, `18px`],
+            ".active": { color: `heading` },
+          }}
+        >
+          {nav.map((item) => {
+            return (
+              <Link
+                key={item?.title}
+                activeClassName="active"
+                sx={(t) => {
+                  return { ...t.styles?.a };
+                }}
+                to={replaceSlashes(`${basePath}/${item.slug}`)}
+              >
+                {item.title}
```

Comment:
To add localization for the word "Блог" in different languages, you would need to manually add the translations in the respective JSON files located under the /locales folder.

Here's an example of how the JSON files might look for the English, Ukrainian, and Russian translations:

*/locales/en/index.json*
```
{
   "Блог": "Blog"
}
```

*/locales/ua/index.json*
```
{
   "Блог": "Блог"
}
```

*/locales/ru/index.json*
```
{
   "Блог": "Блог"
}
```

Then we can use `t()` function like t('Блог'). The t('Блог') would return "Blog". If the user has selected Ukrainian or Russian, it would return "Блог" in both cases.